fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -934,17 +934,13 @@ public void testEnableVersioning() throws Exception {
 
     @Test
     public void testEnableVersioningBinary() throws Exception {
+        logger.debug("testEnableVersioningBinary");
         final String binaryUri = serverAddress + id + "/ds";
         createDatastream(id, "ds", "content");
 
         final String versionsUri = binaryUri + "/" + FCR_VERSIONS;
-        assertEquals("fcr:versions must not exist before enabling versioning",
```

Comment:
because the logic of GET throws a repository exception if the item is not versioned: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersioning.java#L272

Removing them made me save myself from pulling out my hair.  I'll try putting them back in again and see if it behaves any better now that other changes have been made.